pages/tools/cli.js

```diff
@@ -0,0 +1,177 @@
+import GenericLayout from "../../components/layout/GenericLayout";
+import GithubButton from "../../components/buttons/GithubButton";
+
+import {
+  DocumentAddIcon,
+  BadgeCheckIcon,
+  GlobeIcon,
+  CodeIcon,
+} from "@heroicons/react/outline";
+
+import CodeBlock from "../../components/editor/CodeBlock";
+
+const features = [
+  {
+    name: "New files",
+    description: () =>
+      "Use the CLI tool to quickly create new AsyncAPI files. Select from a range of templates (MMQT, WebSockets, Kafka, and more.)",
+    icon: DocumentAddIcon,
+  },
+  {
+    name: "Validate",
+    description: () =>
+      "Validate your AsyncAPI documents with the CLI. Quickly get feedback to verify your AsyncAPI document is within the correct format.",
+    icon: BadgeCheckIcon,
+  },
+  {
+    name: "Open Studio",
+    // eslint-disable-next-line react/display-name
+    description: () => <>Got an AsyncAPI file locally? Run <span className="bg-gray-800 text-gray-100 rounded-sm px-2 py-0.5">asyncapi start studio</span> to open our studio in seconds.</>,
```

Comment:
```suggestion
    description: () => <>Got an AsyncAPI file locally? Run <code className=" px-1 py-0.5 bg-primary-700 text-white rounded font-mono text-sm">asyncapi start studio</code> to open our studio in seconds.</>,
```

and then it will be rendered as ![image](https://user-images.githubusercontent.com/20404945/141136554-97bdde2f-eae6-43c6-bd92-9a12e0b09889.png)`

:)

Reply:
ah cool thanks @magicmatatjahu !